### DISCUSSION TITLE
[Feature] AI Smart search localization

### DISCUSSION THREAD

**Discussion by leocb** [immich-app/immich#11121] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Currently the search works only for english terms. For example:
searching for `chair` works but `Cadeira`(portuguese) does not.
Would it be possible to localize the AI search terms? or maybe do a translation using some API somewhere then pipe the result back to the search?

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by lukasdotcom** [comment]
I know there are [multilingual CLIP models](https://huggingface.co/collections/immich-app/clip-654eaefb077425890874cd07), but I haven't ever used them before so don't know how they'll do. You can change the model in the system settings under Machine Learning Settings > Smart Search > CLIP Model.

**Reply by huydinh282** [reply]
Worked for me in Vietnamese and it was a simple switch. Make sure to do a full smart search job after you switch model

**Reply by leocb** [reply]
Which of these model should I use? There are a lot of options to download and the descriptions on them doesn't help me choose one

**Reply by lukasdotcom** [reply]
According to #5230 [XLM-Roberta-Large-Vit-B-32](https://huggingface.co/immich-app/XLM-Roberta-Large-Vit-B-32) is the best model. I don't know how up to date that is though.

**Reply by lukasdotcom** [reply]
I just tried that model on my instance and searched `Cadeira`. I also tried some other Portuguese words with google translate and they all seemed to work.

**Reply by leocb** [reply]
That's awesome! Thanks!
